docker-compose.yaml

```diff
@@ -1,14 +1,17 @@
-web:
-  image: skyportal/web
-  ports:
-    - "9000:5000"
-    - "9001:5001"
-  links:
-    - db
+version: "3.3"
 
-db:
-  image: postgres:9.6
-  environment:
-    - POSTGRES_USER=skyportal
-    - POSTGRES_PASSWORD=
-    - POSTGRES_DB=skyportal
+services:
+  web:
+    image: skyportal/web
+    ports:
+      - "9000:5000"
+      - "9001:5001"
+    links:
+      - db
+
+  db:
+    image: postgres:9.6
+    environment:
+      - POSTGRES_USER=skyportal
+      - POSTGRES_PASSWORD=latropyks
```

Comment:
Should this be specified in this PR?

Reply:
Well, I just want a placeholder. This should probably be changed anyways, otherwise you'd have to modify `docker-compose.yaml` directly screwing things for the next time you do `git pull`.